cfgov/unprocessed/css/organisms/info-unit-group.less

```diff
@@ -44,16 +44,16 @@
     - cfgov-organisms
 */
 .o-info-unit-group {
+    .content-l_col-1 + .content-l_col-1 {
+        // Adds margin top to info units that follow another
+        margin-top: unit( @grid_gutter-width / @base-font-size-px, em ) !important;
```

Comment:
I've got a PR coming next that's removing it. I couldn't find any reason the original rule in overrides exists